waku/v2/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -11,7 +11,7 @@ import
 import
   ../../../waku_core,
   ../../common,
-  ../../driver,
+  ../../driver_base,
```

Comment:
Nit: I wouldn't call this `_base`. We didn't previously follow this pattern (see e.g. "node" and "builder"). It's clear enough that the builder for the driver is distinct from the driver. I'd either go for `driver.nim` and `driver_builder` or, preferably, `driver.nim` and `builder.nim` as it's fully qualified in import path: `driver/driver.nim` and `driver/builder.nim`. The latter would be the established pattern.